lib/HomeyCompose.js

```diff
@@ -310,7 +311,12 @@ class HomeyCompose {
               });
             }
 
-            const filter = card.$filter || '';
+            let filter = '';
+            if (typeof card.$filter === 'string') {
+              filter = card.$filter;
+            } else if (typeof card.$filter === 'object') {
```

Comment:
Is `typeof object` safe enough? Not sure if it is validated, to prevent e.g. `null` to slip through

Reply:
Luckily `new url.URLSearchParams(null).toString(); === ''` :)